build/ci/.azure-devops-ios-tests-run.yml

```diff
@@ -146,10 +146,20 @@ jobs:
     condition: always()
     retryCountOnTaskFailure: 3
     inputs:
-      PathtoPublish: $(build.artifactstagingdirectory)
+      PathtoPublish: $(build.artifactstagingdirectory)/screenshots
```

Comment:
Actually, let's validate the output because selecting the folder this way may end up not creating the screenshot folder inside the artifacts.

Reply:
Indeed, that may even crash the screenshot comparison tool. I'll adjust the paths used in the tool

Reply:
Or better if there is a way to force Azure to create the directory. Because otherwise I'll need to adjust publishing for other platforms to have consistent path.

Reply:
We can move the logs to a completely separate folders that's not in the default artifacts directory, then publish that.